### ISSUE TITLE
Please maintain helm charts for latest version

### ISSUE THREAD

**Issue by duxinxiao** [danielqsj/kafka_exporter#282]
I have tried helm charts for v1.4.2 v1.4.1 v1.4.0 v1.3.2 branch, none of them can start successfully. (I cant use v1.3.1 cuz kafka_consumergroup_lag metrics not show up in this version)

**Comment by edwin092** [comment]
@duxinxiao could you tell us how you've actually tried out them?

**Comment by duxinxiao** [comment]
![image](https://user-images.githubusercontent.com/6724396/141773578-52405b34-1a17-4dcf-ac6b-ac31b62984ba.png)
this for example, I used to use ref v1.4.2 instead of master(cuz master's charts will not compatible one day, it's better to use static one, to make it static I fork it into my own repository). but v1.4.2's charts have bug in deployment template (lack 'end' line). 
other bug happen when I use v1.4.1's chart & v1.4.1's docker image, v1.4.0's chart & v1.4.0's docker image (relate with new args verbosity). They have different reasons but all not work

**Comment by edwin092** [comment]
If you want, you could raise a PR with the fixes.

**Comment by danielqsj** [comment]
@duxinxiao @edwin092 have a try the chart of master branch

**Comment by edwin092** [comment]
@duxinxiao "end line" issue has been addressed here https://github.com/danielqsj/kafka_exporter/pull/281/files.

**Comment by duxinxiao** [comment]
Yes, I know master fix this issue. but it's not enough only fix it in master, at least it should also fix in each tag release branch. For example, If I use old image version maybe it's not compatible with newest master
